src/target/drivers/storage/mcu_flash.c

```diff
@@ -86,5 +84,8 @@ void MCUFlash_EraseSector(u32 sectorAddress) {
     if (!writable)
         return;
 
+    // Erase two pages in one call
+    // we simulate a 4KB based device
     flash_erase_page(sectorAddress);
+    flash_erase_page(sectorAddress + FLASH_PAGE_SIZE);
 }
```

Comment:
I think this should go into EraseSector4k() or something.  It is non intuitive that it will erase 2 sectors.  What if we set devofs sector size to 2k in the future (which is really what it should be on the at9/10)?

Reply:
will address in a seperate PR to abstract out Disk